treeherder/seta/update_job_priority.py

```diff
@@ -14,7 +14,8 @@
 from treeherder.etl.runnable_jobs import list_runnable_jobs
 from treeherder.etl.seta import (parse_testtype,
                                  valid_platform)
-from treeherder.seta.common import (job_priority_index,
+from treeherder.seta.common import (convert_job_type_name_to_testtype,
```

Comment:
I don't see convert_job_type_name_to_testtype used in the code, can we remove that from the import?

Reply:
I must have eliminated it along the way. Ok.